whitelist.txt

```diff
@@ -224,6 +224,7 @@ callcode
 return
 delegatecall
 create2
+codedeposit
```

Comment:
I would lightly prefer using the underscored spelling: `GAS_CODE_DEPOSIT`. Thoughts?